readme.md

```diff
@@ -33,26 +33,14 @@ In case you forgot to set `--recurse-submodules` during `clone` you can run the
 git submodule update --init
 ```
 
-Prerequisite: 
```

Comment:
We don't want to remove this part of the docs.

Reply:
Got it. I was misunderstanding a bit.